src/classes/app.py

```diff
@@ -39,6 +39,15 @@
 from PyQt5.QtGui import QPalette, QColor, QFontDatabase, QFont, QIcon
 from PyQt5.QtWidgets import QApplication, QStyleFactory, QMessageBox
 
+# QtWebEngineWidgets must be loaded prior to creating a QApplication
+from PyQt5.QtWebEngineWidgets import QWebEngineView
+
+try:
+    # Solution to solve QtWebEngineWidgets black screen caused by OpenGL not loaded
+    from OpenGL import GL
+except ImportError:
+    pass
```

Comment:
Thanks, I'll give this a try! I figured the PyOpenGL was just a work-around until I figured out what was happening. Without it, I only get a black rectangle for the timeline.

Reply:
Mmm. Well, if it works, maybe it doesn't hurt anything to have it there regardless.

My system now **does** have an `OpenGL` module, because `PyOpenGL` was added to the `requirements.txt` when `pipreqs` scanned the source tree, so it's not like it was a really difficult dependency to satisfy.

(Of course, it's only present in the source to be found because of this `import` line, so the whole thing's sort of circular.)